### PR TITLE
Trigger release

### PR DESCRIPTION
**Pull request by pravorskyi**

### CODE REVIEW

File: TriggerRelease/index.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import tl = require('vsts-task-lib/task');
+import url = require('url');
+
+import {EFClient} from "ef-client";
+
+let parseParameters = function(params: string) {
```
Comment (imago-storm) [inline TriggerRelease/index.ts]: How about moving this into the shared library?

File: TriggerRelease/index.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            var message = `Wrong parameters format, either JSON or key=value pairs are required. You have provided: ${params}`;
+            tl.setResult(tl.TaskResult.Failed, message);
+            throw(message);
+        }
+    }
+    return retval;
+}
+
+
+let createFlowRuntimeLink = function(endpoint: string, pipelineId: string, flowRuntimeId: string) {
```
Comment (imago-storm) [inline TriggerRelease/index.ts]: And this one too

File: TriggerRelease/index.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    else {
+        return efClient.release(projectName, releaseName, startingStageName, stagesToRun);
+    }
+}).then((res: any) => {
+    let flowRuntimeId = res.flowRuntime.flowRuntimeId;
+    let runtimeName = res.flowRuntime.flowRuntimeName;
+    let pipelineId = res.flowRuntime.pipelineId;
+    let link = createFlowRuntimeLink(efBaseUrl, pipelineId, flowRuntimeId);
+    console.log("Pipeline run succeeded, runtime name is " + runtimeName);
```
Comment (imago-storm) [inline TriggerRelease/index.ts]: I guess it should be release run

File: TriggerRelease/index.ts (hunk 1 above)
Comment (pravorskyi) [reply TriggerRelease/index.ts]: Done

File: TriggerRelease/index.ts (hunk 2 above)
Comment (pravorskyi) [reply TriggerRelease/index.ts]: Done

File: TriggerRelease/index.ts (hunk 3 above)
Comment (pravorskyi) [reply TriggerRelease/index.ts]: Done

**Review by imago-storm — APPROVED** [review]

**Review by imago-storm — APPROVED** [review]

**Pull request merged**